package.json

```diff
@@ -1828,7 +1828,7 @@
         "command": "dotnet.generateAssets",
         "title": "%command.dotnet.generateAssets.currentProject%",
         "category": ".NET",
-        "enablement": "dotnet.server.activationContext == 'Roslyn' || dotnet.server.activationContext == 'OmniSharp'"
+        "enablement": "dotnet.server.activationContext == 'RoslynDevKit' || dotnet.server.activationContext == 'Roslyn' || dotnet.server.activationContext == 'OmniSharp'"
```

Comment:
Should be fine to delete the enablement clause entirely - Roslyn/RoslynDevKit and OmniSharp are the only options

Reply:
We still want an enablement command, but maybe there is a better one -- we only want the command to be enabled when a language server is running. See [this commit](https://github.com/dotnet/vscode-csharp/commit/357cefbdb9921831b0a8c93bcc583829d80a7fcc).

Reply:
Ah I missed that.  Likely we should create a different 'server activated context' (would help with the perf work I was doing before), but I won't block on that.